web-team/php-challenges.md

```diff
@@ -0,0 +1,48 @@
+# PHP challenge #
```

Comment:
The file exists in the `add_readme` branch but not `master`. Part of this PR is to merge (fast-forward) everything into `master`, because it seems like the `add_readme` and `android` branches are no longer needed.